### PR TITLE
remove skipped tests

### PR DESCRIPTION
**Pull request by cfaur09**
## Proposed Changes
- All tests of the form `e2e.spec` -> `.spec` have been modified and mocked to allow a much faster execution of all tests and at the same time to cover much more of the functionality of the services
- All unit tests will be found in the /unit/services directory and all tests are unitary, so they do not depend on any source (db, elastic, etc).

## How to test
- run command: `npm run test`

### CODE REVIEW

File: src/test/integration/services/transaction.get.e2e-spec.ts [hunk 1]
```diff
@@ -85,6 +84,9 @@ describe('Transaction Get Service', () => {
           throw new HttpException({ status: HttpStatus.BAD_GATEWAY, message: "BAD GATEWAY" }, HttpStatus.BAD_GATEWAY);
         }));
 
+      jest.spyOn(transactionGetService['logger'], 'error').mockImplementation(() =>
+        "Unexpected error when getting transaction from elastic, hash '51ffbf3d27e06fd509c510ef0ff1ea7329359dba89c05aeec333de52a405664d'");
```
Comment (claudiulataretu) [inline src/test/integration/services/transaction.get.e2e-spec.ts:88]: this should throw an error?

File: src/test/unit/services/collections.spec.ts [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+import { TokenAssetStatus } from "src/endpoints/tokens/entities/token.asset.status";
+import { VmQueryService } from "src/endpoints/vm.query/vm.query.service";
+
+describe('CollectionService', () => {
+  let service: CollectionService;
+  let indexerService: IndexerService;
+  let esdtAddressService: EsdtAddressService;
+  let assetsService: AssetsService;
+
+  const indexerCollectionMock = {
```
Comment (claudiulataretu) [inline src/test/unit/services/collections.spec.ts:26]: based on last discussion on mocks, this should be a stub :)

File: src/test/unit/services/collections.spec.ts [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+    api_isVerified: true,
+    api_nftCount: 131,
+  };
+
+  beforeEach(async () => {
+    const moduleRef = await Test.createTestingModule({
+      providers: [
+        CollectionService,
+        {
+          provide: IndexerService, useValue: {
```
Comment (claudiulataretu) [inline src/test/unit/services/collections.spec.ts:71]: put `useValue` on different line

File: src/test/unit/services/esdt.address.spec.ts [hunk 4]
```diff
@@ -0,0 +1,347 @@
...
+  let collectionService: CollectionService;
+  let cacheService: CacheService;
+  let metricsService: MetricsService;
+  let protocolService: ProtocolService;
+  beforeEach(async () => {
+    const moduleRef = await Test.createTestingModule({
+      providers: [
+        EsdtAddressService,
+        {
+          provide: IndexerService, useValue: {
```
Comment (claudiulataretu) [inline src/test/unit/services/esdt.address.spec.ts:33]: put `useValue` on new line

File: src/test/unit/services/scresults.spec.ts [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+      esdtValues: ['123416452926529462'],
+      operation: 'ESDTTransfer',
+    }];
+
+    const mockTransactionAction = {
+      category: 'esdtNft',
+      name: 'transfer',
+      description: 'Transfer',
+      arguments: {
+        transfers: [[Object]],
```
Comment (claudiulataretu) [inline src/test/unit/services/scresults.spec.ts:163]: Object should be changed with actual data

Comment (cfaur09) [reply src/test/unit/services/scresults.spec.ts]: fixed

File: src/test/unit/services/esdt.address.spec.ts (hunk 4 above)
Comment (cfaur09) [reply src/test/unit/services/esdt.address.spec.ts]: fixed

File: src/test/unit/services/collections.spec.ts (hunk 2 above)
Comment (cfaur09) [reply src/test/unit/services/collections.spec.ts:26]: will be change in a stub

File: src/test/integration/services/transaction.get.e2e-spec.ts (hunk 1 above)
Comment (cfaur09) [reply src/test/integration/services/transaction.get.e2e-spec.ts:88]: This throw an error
```
jest
        .spyOn(IndexerService.prototype, "getTransaction")
        // eslint-disable-next-line require-await
        .mockImplementation(jest.fn(async (_: string) => {
          throw new HttpException({ status: HttpStatus.BAD_GATEWAY, message: "BAD GATEWAY" }, HttpStatus.BAD_GATEWAY);
        }));
```
        
The highlighted code from this comment, it refers to the fact that I also treat the error log that appears after the execution of the test

**Review by tanghel — APPROVED** [review]

**Pull request merged**